src/javascript/app/pages/resources/economic_calendar/economic_calendar.js

```diff
@@ -8,6 +10,13 @@ const EconomicCalendar = (() => {
 
         const curr_language = getLanguage().toLowerCase();
 
+        BinarySocket.wait('website_status', 'authorize', 'landing_company').then(() => {
+            const $footer_el = $('.calendar-footer');
+            if (isEuCountry() && $footer_el) {
+                $footer_el.setVisibility(1);
+            }
```

Comment:
can be simplified to `$('.calendar-footer').setVisibility(isEuCountry())`, since jQuery already checks that element exists and we don't need to do that check again.

Reply:
Not sure if it helps much (as I couldn't measure the difference on my end),but it does make more sense to move the visibility toggle to `populateCalendar` function once presumably the script has finished loading.